src/NuGet/Microsoft.Net.Compilers/CompilerArtifacts.targets

```diff
@@ -56,10 +56,16 @@
       <_NoOptimizationData Include="$(ArtifactsBinDir)csi\$(Configuration)\net472\System.Threading.Tasks.Extensions.dll"/>
       <_NoOptimizationData Include="$(ArtifactsBinDir)csi\$(Configuration)\net472\System.Buffers.dll"/>
       <_NoOptimizationData Include="$(ArtifactsBinDir)csi\$(Configuration)\net472\System.Memory.dll"/>
-      <_NoOptimizationData Include="$(ArtifactsBinDir)csi\$(Configuration)\net472\System.Numerics.Vectors.dll"/>
+      <_NoOptimizationData Include="$(ArtifactsBinDir)csi\$(Configuration)\net472\System.Numerics.Vectors.dll"/>      
```

Comment:
If the DLL occurs twice in our deployment, say the csc and csci directory, does it need to be listed once or twice here?

Reply:
Once.

Reply:
This target defines a set of compile artifacts. The calling project is deploying them. We have multiple projects that deploy them.